src/main/java/com/mixpanel/android/viewcrawler/EditState.java

```diff
@@ -119,15 +126,20 @@ private void applyChangesFromList(View rootView, List<ViewVisitor> changes) {
             final int size = changes.size();
             for (int i = 0; i < size; i++) {
                 final ViewVisitor visitor = changes.get(i);
-                final EditBinding binding = new EditBinding(rootView, visitor, mUiThreadHandler);
-                mCurrentEdits.add(binding);
+                try {
+                    final EditBinding binding = new EditBinding(rootView, visitor, mUiThreadHandler);
+                    mCurrentEdits.add(binding);
+                } catch (ViewVisitor.LayoutUpdateException e) {
+                    mEditErrorMessage.sendErrorMessage(e.getErrorInfo());
```

Comment:
This seems like it may be to early to catch the error- we'll continue visiting every time the system updates the global view layout, and could throw in there, too.